systemctl_mqtt/__init__.py

```diff
@@ -388,7 +388,7 @@ async def _dbus_signal_loop_unit(  # pylint: disable=too-many-arguments
         unit_name=unit_name
     )
     ((_, last_active_state),) = await unit_proxy.Get(property_name="ActiveState")
-    await mqtt_client.publish(topic=active_state_topic, payload=last_active_state)
+    await mqtt_client.publish(topic=active_state_topic, payload=last_active_state, retain=True)
```

Comment:
just for reference: in my [last comment](https://github.com/fphammerle/systemctl-mqtt/issues/201#issuecomment-2763408780), I missed that line:
 > systemctl-mqtt should probably be adapted to report the state of all monitored units on systemctl-mqtt's startup

I thought systemctl-mqtt does not report the status of monitored units on startup, but apparently it does already